setup.py

```diff
@@ -50,8 +50,12 @@
             'pytest>=2.7',
             'requests-mock',
             'xmltodict',
+            'shopify_python==0.1.2',
         ],
     },
+    dependency_links=[
+        'git+https://github.com/Shopify/shopify_python.git@v0.1.2#egg=shopify_python-0.1.2',
```

Comment:
In the long-term we shouldn't use `dependency_links` because they're deprecated. We should instead deploy `shopify_python` to pypi or switch to using a `dev_requirements.txt` for our CI/test env (although separating dependencies into separate files makes it harder to find them).